### DISCUSSION TITLE
Features for MF 2.x.x (new release)

### DISCUSSION THREAD

**Discussion by char-star** [ONDC-Official/ONDC-FIS-Specifications#394] [Ideas]
I'm gathering thoughts on list of features that need support in the next version of MF protocol. Requesting all participants to share thoughts, so we can together prioritize.

### March 2025
1. Ability to place order in multiple schemes together (cart functionality) - SAME AMC & DIFFERENT AMCs (Refer: https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/400)
2. Support for direct schemes - RIAs, EOPs as buyer apps (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/407)
3. Payment experience fixes (#436)
4. Support for additional transaction types
    - Redemption by units, redeem all (#401)
 5. Minor fixes/improvements from previous version (#454)

### April 2025
6. Support for additional transaction types
    - SWP (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/481)
    - Switch, STP (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/408)
    - Instant redemption (https://github.com/ONDC-Official/ONDC-FIS-Specifications/pull/494)
    - SIP: pay for first instalment instantly (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/402)
    - SIP: register SIP without mandate, add mandate later (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/402)
    - Lumpsum + SIP (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/403)
    - Lumpsum w/ payment via enach mandate (for large payments) - Possible in v2.0.0
7. SIP/STP/SWP modifications (https://github.com/ONDC-Official/ONDC-FIS-Specifications/pull/486)
8. Support for NRI as investor types (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/404)
9. Refund visibility in case of payment failures/ fulfillment failures (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/406)
10. Support for transactions against external folios (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/489)
11. Ability to create zero balance folio (https://github.com/ONDC-Official/ONDC-FIS-Specifications/pull/487)
12. Buyer app collecting payments (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/490)
13. Nomination changes - include video capture, additional fields, max. 10 nominees (https://github.com/ONDC-Official/ONDC-FIS-Specifications/pull/493)

### Others
14. Support for Minors
15. Support for Corporates/Non-individuals as investor types (https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/405)
16. Support biometric aadhaar kyc
17. Support for Brokers as buyer apps - solve for demat purchases, demat redemptions
18. Support for non-financial transactions
19. Ability to fetch investor holdings

**Comment by girishgupta211** [comment]
Here will we support switch from both ways ie regular to direct and direct to regular funds

**Comment by vjn-validus** [comment]
Still wary of different AMCs in the same cart. Please confirm how could this be achieved without violating the regulatory norms - Realization based NAV and Non-Pooling of Funds.

**Comment by ShubhangaAcharya** [comment]
Requesting considering https://github.com/ONDC-Official/ONDC-FIS-Specifications/issues/460 also as part of the protocol changes.
